src/Claims/AggregateParser.php

```diff
@@ -15,7 +15,7 @@ public function unpack(ClientInterface $client, array $claims): array
         $claimSources = $claims['_claim_sources'] ?? null;
         $claimNames = $claims['_claim_names'] ?? null;
 
-        if (! is_array($claimSources)) {
+        if (! array_key_exists('_claim_sources', $claims) || ! is_array($claimSources)) {
```

Comment:
Why this check? If `_claim_sources` does not exist it will be null as for the check 3 lines before

Reply:
I think (it been a minute) that these extra checks were introduced to stop psalm from grumbling. 

I'll verify.